config/locales/diaspora/en.yml

```diff
@@ -884,6 +884,45 @@ en:
         Hoping to see you again,
 
         The diaspora* email robot!
+  api:
+    openid_connect:
+      authorizations:
+        new:
+          redirection_message: "Are you sure you want to give access to %{redirect_uri}?"
+          access: "%{name} requires access to:"
+          no_requirement: "%{name} requires no permissions"
```

Comment:
As this sentence is displayed on the authorizations page with the application, I though present made more sense. As the application *still* requires no permission but could require in the future.